src/pages/tabs/tabs.ts

```diff
@@ -236,7 +239,7 @@ export class TabsPage {
 
   private async fetchAllWalletsStatus() {
     this.logger.debug('Fetching All Wallets and calculate Total Amount');
-    const wallets = this.profileProvider.getWallets();
+    const wallets = this.profileProvider.wallet;
```

Comment:
Not sure. But that way I think you are getting the hidden wallets too.

Reply:
Ok, I'll add a filter. Thanks!